bee-protocol/src/protocol/helpers.rs

```diff
@@ -133,21 +117,19 @@ impl Protocol {
 
     // Solidifier
 
-    pub fn trigger_transaction_solidification(hash: Hash, index: MilestoneIndex) {
-        if let Err(e) = Protocol::get()
-            .transaction_solidifier_worker
-            .unbounded_send(TransactionSolidifierWorkerEvent(hash, index))
-        {
-            warn!("Triggering transaction solidification failed: {}.", e);
-        }
-    }
-
-    pub fn trigger_milestone_solidification() {
-        if let Err(e) = Protocol::get()
-            .milestone_solidifier_worker
-            .unbounded_send(MilestoneSolidifierWorkerEvent)
-        {
-            warn!("Triggering milestone solidification failed: {}.", e);
+    pub fn trigger_milestone_solidification(target_index: MilestoneIndex) {
+        if let Some(target_hash) = tangle().get_milestone_hash(target_index) {
+            if !tangle().is_solid_transaction(&target_hash) {
+                traversal::visit_parents_depth_first(
+                    tangle(),
+                    target_hash,
+                    |hash, _, metadata| {
+                        !metadata.flags.is_solid() && !Protocol::get().requested_transactions.contains_key(&hash)
```

Comment:
This can still traverse to previous milestones

Reply:
Maybe you can quickly try with your ((((lisp)))) script

Reply:
sure